File: CODIGOS/codeabbey140.r
countOnes <- function(n) {
  count <- 0
  for (i in 1:n) {
    for (j in 1:n) {
      if (i == j) {
        count <- count + 1
      }
    }
  }
  return(count)
}

n = as.integer(readline("Enter the size of the matrix: "))
print(countOnes(n))

File: CODIGOS/codeabbey140.lua
function countOnes(n)
    count = 0
    for i = 1, n do
      for j = 1, n do
        if i == j then
          count = count + 1
        end
      end
    end
    return count
  end
  
  n = tonumber(io.read())
  print(countOnes(n))
  